docs/templates_system/containers_and_widgets.rst

```diff
@@ -14,7 +14,7 @@ Every container can have default parameters and content, and can be hidden when
 The container twig tag keeps HTML syntax always up to date with JavaScript live management expectations.
 Container default values can be overridden by widgets.
 
-What is a Widget?
+What is a WidgetModel?
```

Comment:
why WdgetModel? It's a general definition of the widget inside the Web Publisher